docs: initial start of the maintainership docs under the bounty program (#1894)
asyncapi/community: docs/maintainership-guide/general-information/contributor/benefits.md

---
title: Contributor Benefits
weight: 90
---

> You might be thinking isn’t access to world-class mentorship and a generous stipend enough? While those are core parts of the AMP experience, the **AsyncAPI Mentorship Program offers so much more**.

Reviewer: ```suggestion
You might be thinking, isn’t access to world-class mentorship and a generous stipend enough? While those are core parts of the AMP experience, the **AsyncAPI Mentorship Program offers so much more**.
```

Here’s what contributors can expect to gain:

## Personalized  Mentorship

As an AMP contributor, you’ll have the unique opportunity to learn directly from the core team behind AsyncAPI, including Technical Steering Committee members, Ambassadors, and experienced maintainers. These mentors are deeply committed to your personal growth and to nurturing the community as a whole.

> ## Real world Experience

Reviewer: ```suggestion
## Real World Experience
```

> We believe the best learning happens when you're solving real problems. That’s why AMP contributors work on production-grade, high-impact projects used by real teams and companies. Your work won’t sit in a sandbox, it will be merged, deployed, and used in the real world, often during the program itself.

Reviewer: ```suggestion
We believe that the best learning occurs when you're solving real-world problems. That’s why AMP contributors work on production-grade, high-impact projects used by real teams and companies. Your work won’t sit in a sandbox; it will be merged, deployed, and used in the real world, often during the program itself.
```

## Community Recognition:

> Your efforts won’t go unnoticed. AMP contributors gain visibility and credibility within the AsyncAPI ecosystem and broader open source space. You may even be invited to speak at AsyncAPI events, share your journey, inspire others, and showcase your project to a global audience.

Reviewer: ```suggestion
Your efforts won’t go unnoticed. AMP contributors gain visibility and credibility within the AsyncAPI ecosystem and the broader open source space. You may even be invited to speak at AsyncAPI events, share your journey, inspire others, and showcase your project to a global audience.
```

## Career-Boosting Visibility:

> AMP is a launchpad. Many alumni have gone on to land internships, full-time roles, or speaking engagements thanks to the skills built during the program. Contributors receive public endorsements, mentorship reviews, and a certificate of completion, all of which strengthen your portfolio and résumé.

Reviewer: ```suggestion
AMP is a launchpad. Many alumni have gone on to secure internships, full-time roles, or speaking engagements thanks to the skills they developed during the program. Contributors receive public endorsements, mentorship reviews, and a certificate of completion, all of which strengthen your portfolio and résumé.
```
